curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/65f03d9f92eac9183a4d3281.md

```diff
@@ -0,0 +1,37 @@
+---
+id: 65f03d9f92eac9183a4d3281
+title: Step 2
+challengeType: 20
+dashedName: step-2
+---
+
+# --description--
+
+A vector can be defined by two coordinates, `x` and `y`, in the Euclidean plane. The distance between the origin of the axes and the point `(x, y)` will be its length, or norm. And the vector direction will point towards `(x, y)`.
+
+Within the `Vector` class, create an `__init__` method and give it three parameters, `self`, `x`, and `y`.
```

Comment:
As the project targets special methods, it would it be good to mentionin that `__init__` is a special method.